crates/pgsrv/src/handler.rs

```diff
@@ -699,7 +699,8 @@ where
             ExecutionResult::Rollback => Self::command_complete(conn, "ROLLBACK").await?,
             ExecutionResult::WriteSuccess => Self::command_complete(conn, "INSERT").await?,
             ExecutionResult::CopySuccess => Self::command_complete(conn, "COPY").await?,
-            ExecutionResult::DeleteSuccess => Self::command_complete(conn, "DELETE").await?,
+            ExecutionResult::DeleteSuccess { .. } => Self::command_complete(conn, "DELETE").await?,
+            ExecutionResult::UpdateSuccess { .. } => Self::command_complete(conn, "UPDATE").await?,
```

Comment:
Since we have the numbers now, we can include those here. Should look like "DELETE 1" or "UPDATE 1".